### DISCUSSION TITLE
혹시 일괄 다운로드 된 txt파일에서 추출이 가능한가요?

### DISCUSSION THREAD

**Discussion by FeelsBotMan** [josw123/dart-fss#138] [Ideas]
상장사 모두의 보고서들을 가져올려고 해봤는데 시간이 너무 오래 걸리더라고요

혹시 https://opendart.fss.or.kr/disclosureinfo/fnltt/dwld/main.do 에서 일괄 다운로드한 txt파일에서

추출하면 좀 더 빠를 것 같은데 혹시 지금도 지원하는 기능인가요?

**Comment by josw123** [comment]
안녕하세요.

올려주신 텍스트 파일을 이용한 추출은 현재 지원되지 않는 기능입니다.
제작 당시 일괄다운로드에 있는 내용이 부실하여 관련 코드를 작성하지 않았습니다만,
한번 검토해보도록 하겠습니다.
